### PR TITLE
various minor cleanup items

### PR DESCRIPTION
**Pull request by ellisonanne**

### CODE REVIEW

**Comment by smarthi** [comment]
+1 lgtm

File: src/main/java/org/apache/pirk/encryption/PrimeGenerator.java [hunk 1]
```diff
@@ -18,15 +18,15 @@
...
-
 import java.math.BigDecimal;
 import java.math.BigInteger;
 import java.util.HashMap;
 import java.util.Random;
 
+import org.apache.pirk.utils.SystemConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
```
Comment (tellison) [inline src/main/java/org/apache/pirk/encryption/PrimeGenerator.java]: I wonder if we are getting into an IDE commit-fest here, where one IDE thinks the imports should be ordered one way, and another brand of IDE thinks differently.

**Comment by ellisonanne** [comment]
Yes, I ran 'organize imports' in Eclipse

**Comment by tellison** [comment]
+1 to merge

**Pull request closed without merging**